getting_started/step_by_step/index.rst

```diff
@@ -56,7 +56,7 @@ With that out of the way, let's get started by
    *
 
 .. _editor introduction: https://docs.godotengine.org/en/stable/getting_started/step_by_step/intro_to_the_editor_interface.html
-.. _nodes documentation: https://docs.godotengine.org/en/stable/getting_started/step_by_step/scenes_and_nodes.html
+.. _key concepts: https://docs.godotengine.org/en/stable/getting_started/introduction/key_concepts_overview.html
```

Comment:
Thank you for this catch.
I think the new link should be https://docs.godotengine.org/en/stable/getting_started/step_by_step/nodes_and_scenes.html#nodes 
Indeed they seem to have renamed the page.

Reply:
Thank you and all the team for this amazing work! Just starting to learn Godot & Escoria and it is quite impressive.